org.musescore.MuseScore.yaml

```diff
@@ -66,15 +66,26 @@ modules:
     sources:
       - type: git
         url: https://github.com/steinbergmedia/vst3sdk.git
-        tag: v3.7.12_build_20
-        commit: cc2adc90382dded9e347caf74e4532f1458715db
+        tag: v3.8.0_build_66
+        commit: 9fad9770f2ae8542ab1a548a68c1ad1ac690abe0
         dest: vst3sdk
+        x-checker-data:
+          type: git
+          tag-pattern: ^v(\d+.\d+.\d+_build_\d+)$
```

Comment:
```suggestion
```

1. it's pinned
2. you are not the maintainer, this create burden (spam) on the maintainer.

Reply:
Ok! I was only trying to help.